src/pyseekdb/client/client_base.py

```
@@ -2562,19 +2561,16 @@ def _collection_hybrid_search(
-
-        logger.debug(f"Executing query SQL: {query_sql}")
-
-        # Execute the returned SQL query
-        result_rows = self._execute_query_with_cursor(conn, query_sql, [], use_context_manager)
+        # Parse the search result JSON
+        search_result_json = rows[0]["search_result"]
+        if isinstance(search_result_json, str):
+            search_result = json.loads(search_result_json)
+        else:
+            search_result = search_result_json
+        logger.debug(f"Search result received from DBMS_HYBRID_SEARCH.SEARCH")
```

Ruff (0.14.14)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)